aten/src/ATen/native/native_functions.yaml

```diff
@@ -13170,6 +13170,11 @@
   dispatch:
     CUDA: flash_scaled_dot_product_attention
 
+- func: _efficient_attention(Tensor query, Tensor key, Tensor value, bool compute_log_sumexp=False, Tensor? attn_bias=None, float p=0.0, bool causal=False) -> (Tensor, Tensor)
```

Comment:
native functions for some reason doesn't let you return `int` inside of a tuple.